bin/hawtio-proxy.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This script does all the proxy magic we need to access hawtio on a cluster
+NGINX_PORT=${NGINX_PORT:-8085}
+FORWARD_PORT=${FORWARD_PORT:-9095}
+pod=$1
+if [ $# -lt 1 ]; then
+  echo "Usage: $0 <pod>"
+  exit 1
+fi
+if [ -z "$(which jq)" ]; then
+  echo "jq not found (hint: sudo dnf install -y jq)"
```

Comment:
Do you want to bomb out here?  As it is, the script will keep running and then fail when `jq` can't be found on line 16.